dotCMS/src/integration-test/java/com/dotcms/datagen/UserDataGen.java

```diff
@@ -22,6 +23,8 @@
     private String lastName = "testLastName" + currentTime;
     private String emailAddress = "testEmailAddress@" + currentTime + ".com";
     private String password = String.valueOf(currentTime);
+    private String skinIdentifier = UUIDGenerator.generateUuid();
+    private String companyId = UUIDGenerator.generateUuid();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field companyId has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)